### ISSUE TITLE
Update: Section 13.5 - p. 536 - Update Nonblocking Collectives

### ISSUE THREAD

**Issue by wesbland** [mpi-forum/mpi-issues#248]
## Problem

Under Coordination.  This text predates the addition of nonblocking collective routine and needs to be corrected.

**Comment by wesbland** [comment]
Needs discussion to scope.

* Editor - 0
* Chapter committee - 3
* Errata - 3
* Full Proposal - 1

**Comment by wesbland** [comment]
@wgropp It's unclear what's meant by this issue since we can't find the reference.

**Comment by wgropp** [comment]
I think this was 13.4.5, the split IO collectives, which was written for MPI-2 when non-blocking collectives were rejected for MPI-2. This is now 14.4.5.  Now that there are nonblocking IO collectives such as MPI_FILE_IWRITE_ALL, these routines are obsolescent. At the very least, that bit of history should be added, and perhaps a note to prefer the newer, fully collective, routines.

**Comment by dholmes-epcc-ed-ac-uk** [comment]
IMHO, adding a historical note might be chapter committee level (depending on exact wording, which suggests oversight/voting would be useful), but adding a preference and/or advice to prefer is a slightly bigger deal and that should be subject to a vote. Our rules give two routes for that: errata (but it is not wrong per se), or full proposal. Both target MPI-4.1 at the earliest now due to the advanced timeline for MPI-4.0, right?

Also, there are future proposals (e.g. https://github.com/mpi-forum/mpi-issues/issues/78 to add nonblocking file open and close, and interest in adding persistent I/O read/write routines), which give us clear future opportunities to fix this (meaning the preference part, but the historical note - if added earlier - would probably need updating by these future changes).

**Comment by wesbland** [comment]
I agree that we've probably missed 4.0 for this. That's why we removed that label during the call yesterday.
